Packs/Netskope/Integrations/NetskopeEventCollector/NetskopeEventCollector.py

```diff
@@ -116,18 +116,20 @@ def dedup_by_id(last_run: dict, results: list, event_type: str, limit: int):
     events = []
     last_run_ids = set(last_run.get(f'{event_type}-ids', []))
     # Sorting the list to Ascending order according to the timestamp (old one first)
-    for event in list(reversed(results))[:limit]:
+    sorted_list = list(reversed(results))
```

Comment:
Why do you need the casting to list? reversed returned an iterator that you can use in the for loop.

Reply:
See my comment before - I think it is easier and more clear to work with a list rather than the iterator. 
I can change that if you think it will be better